src/reducers/Board/index.ts

```diff
@@ -107,12 +110,17 @@ export const openCellAction = (x: number, y: number): BoardActionTypes => {
   };
 };
 
-export const toggleFlagAction = (x: number, y: number): BoardActionTypes => {
+export const toggleFlagAction = (
+  x: number,
+  y: number,
+  isFlagged: boolean
+): BoardActionTypes => {
   return {
     type: ActionTypes.TOGGLE_FLAG,
     payload: {
       x: x,
       y: y,
+      isFlagged: isFlagged,
     },
```

Comment:
わかってるかもしれないけど、
keyとvalueの名前が同じ時に、省略できるのは知っといたら良いかもね
```suggestion
    payload: {
      x,
      y,
      isFlagged,
    },
```